third_party/cuquantum/cuquantum_configure.bzl

```diff
@@ -0,0 +1,220 @@
+"""Setup cuQuantum as external dependency"""
```

Comment:
How does this file get used when building for cuQuantum? Specifically, what outside this file triggers the `cuquantum_configure` rule?

Reply:
Argh, that's what I get for marking `WORKSPACE` as reviewed too early...

LGTM.

Reply:
thank you! I applied your feedback. after that, please LGTM this PR :D